### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by Lupie**
Rebase

### CODE REVIEW

**Comment by Lupie** [comment]
@blampe

**Comment by blampe** [comment]
:ship:

File: mkdocs.yml [hunk 1]
```diff
@@ -12,9 +12,5 @@ pages:
 - Go Guide: go-guide.md
 - Node Guide: node-guide.md
 - Python Guide: python-guide.md
-theme: readthedocs
 
-markdown_extensions:
-  - toc:
-      permalink: 
-  - admonition:
+theme: cosmo
```
Comment (breerly) [inline mkdocs.yml]: You switched the theme? Nice! Can we get a screenshot before you merge? tanks :)

Comment (Lupie) [inline mkdocs.yml]: Yep, I have a couple.

Comment (breerly) [inline mkdocs.yml]: You can upload them here by dragging the screenshot to the comment box!

**Pull request merged**